TwitchClient/GamesListViewController.swift

```diff
@@ -1,108 +0,0 @@
-//
-//  GamesListViewController.swift
-//  TwitchClient
-//
-//  Created by Ahmed kotb on 4/7/17.
-//  Copyright © 2017 Ahmed Kotb. All rights reserved.
-//
-
-import UIKit
-import SVProgressHUD
-
-class GamesListViewController: UIViewController, WebserviceManagerDelegate, UITableViewDelegate, UITableViewDataSource {
-
-    
-    @IBOutlet weak var gamesTableView: UITableView!
-    
-    
-    var games = [Game]()
-   
-    var selectedGame = Game()
-    
-    override func viewDidLoad() {
-        super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
-        
-        self.gamesTableView.isHidden = true
-        SVProgressHUD.show()
-        
-        self.getTopGames()
-        
-    }
-   
-    func getTopGames()
```

Comment:
Same thing applies on all the below networking code, of course. But I'm gonna comment on the implementation with the understanding that it still doesn't belong here even if changed.